client/src/app/admin/panel/users/admin-panel-custom-field-form.js

```diff
@@ -63,7 +63,7 @@ class AdminPanelCustomFieldForm extends React.Component {
 
         return (
             <Message showMessage={showErrorMessage} onCloseMessage={this.onCloseMessage.bind(this, "showErrorMessage")} type="error">
-                {error}
+                {i18n(error)}
```

Comment:
It's great you noticed this one, so let's create the entry for `INVALID_CUSTOM_FIELD_OPTIONS` in `en.js` with the message.